tests/cluster/tests/16-transactions-on-replica.tcl

```diff
@@ -0,0 +1,48 @@
+# Check basic transactions on a replica.
+
+source "../tests/includes/init-tests.tcl"
+
+test "Create a primary with a replica" {
+    create_cluster 1 1
+}
+
+test "Cluster should start ok" {
+    assert_cluster_state ok
+}
+
+set primary [Rn 0]
+set replica [Rn 1]
```

Comment:
are can you know that this is the replica?
aren't you suppose to use something like this?
```
        set replica_id [get_one_of_my_replica $master_id]
```

Reply:
Not a TCL cluster tests expert

But I think this is a side affect of the cluster creation method.
I have followed the `05-slave-selection.tcl` test example in which `create_cluster 5 10` creates and array of:
* 0-4 : primaries
* 5-9 : 1st replicas
* 10-14 : 2nd replica

Reply:
ohh, i now see you used `create_cluster 1 1`. so there are really no other options.